service.xbmc.versioncheck/resources/lib/version_check/distro/distro.py

```diff
@@ -0,0 +1,1216 @@
+# Copyright 2015,2016,2017 Nir Cohen
```

Comment:
I assume that this file comes from https://github.com/nir0s/distro/blob/master/distro.py? 

Assuming that more people might need this functionality (espcially in Python 3.8+) it would make sense to please submit a proper `script.module.distro` add-on? @ronie  you suggested to just include it. What would be the best?

Reply:
I think this add-on is included by default in Kodi, so moving it to a script.module will just increase complexity for no reason. Since it is a single file, it's say to keep it bundled

Reply:
i'm not sure what the best practice is here..

for instance, if we want to ship kodi with the new python based scrapers, most of them have several dependencies as well which we would have to include in kodi.

Reply:
I'm happy to proceed with either method once it's decided.